.ci/integrationTestEC.groovy

```diff
@@ -202,7 +83,43 @@ pipeline {
   }
 }
 
+def runAllTests(){
+  def tests = [
+    'go',
+    'dotnet',
+    'java',
+    'nodejs',
+    'php',
+    'python',
+    'ruby',
+    'rum',
+    'all'
+  ]
```

Comment:
What do you think to use https://github.com/elastic/apm-pipeline-library/blob/main/vars/agentMapping.groovy ?

it requires to support a new step to list all the APM agents anyway

Reply:
none of those has the same list, it is a list on two places, we can move it but in a follow up.